src/vs/editor/contrib/documentSymbols/outlineTree.ts

```diff
@@ -273,3 +273,146 @@ export class OutlineDataSource implements IDataSource<OutlineModel, OutlineItem>
 		return values(element.children);
 	}
 }
+
+
+export const OUTLINE_DEFAULT_SYMBOL_FOREGROUND = registerColor('outlineDefaultSymbol.foreground', {
+	dark: foreground,
+	light: foreground,
+	hc: foreground
+}, localize('outlineDefaultSymbol.foreground', 'The foreground color for default symbols. These symbols appear in the outline, breadcrumb, and suggest widget.'));
+
+export const OUTLINE_METHOD_SYMBOL_FOREGROUND = registerColor('outlineMethodSymbol.foreground', {
+	dark: '#B180D7',
+	light: '#652D90',
+	hc: '#B180D7'
+}, localize('outlineMethodSymbol.foreground', 'The foreground color for method symbols. These symbols appear in the outline, breadcrumb, and suggest widget.'));
+
+export const OUTLINE_CLASS_SYMBOL_FOREGROUND = registerColor('outlineClassSymbol.foreground', {
+	dark: '#EE9D28',
+	light: '#D67E00',
+	hc: '#EE9D28'
+}, localize('outlineClassSymbol.foreground', 'The foreground color for class symbols. These symbols appear in the outline, breadcrumb, and suggest widget.'));
+
+export const OUTLINE_ENUMERATOR_SYMBOL_FOREGROUND = registerColor('outlineEnumeratorSymbol.foreground', {
+	dark: '#EE9D28',
+	light: '#D67E00',
+	hc: '#EE9D28'
+}, localize('outlineEnumeratorSymbol.foreground', 'The foreground color for enumerator symbols. These symbols appear in the outline, breadcrumb, and suggest widget.'));
+
+export const OUTLINE_ENUMERATOR_MEMBER_SYMBOL_FOREGROUND = registerColor('outlineEnumeratorMemberSymbol.foreground', {
+	dark: '#75BEFF',
+	light: '#007ACC',
+	hc: '#75BEFF'
+}, localize('outlineEnumeratorMemberSymbol.foreground', 'The foreground color for enumerator member symbols. These symbols appear in the outline, breadcrumb, and suggest widget.'));
+
+export const OUTLINE_EVENT_SYMBOL_FOREGROUND = registerColor('outlineEventSymbol.foreground', {
+	dark: '#EE9D28',
+	light: '#D67E00',
+	hc: '#EE9D28'
+}, localize('outlineEventSymbol.foreground', 'The foreground color for event symbols. These symbols appear in the outline, breadcrumb, and suggest widget.'));
+
+export const OUTLINE_FIELD_SYMBOL_FOREGROUND = registerColor('outlineFieldSymbol.foreground', {
+	dark: '#75BEFF',
+	light: '#007ACC',
+	hc: '#75BEFF'
+}, localize('outlineFieldSymbol.foreground', 'The foreground color for field symbols. These symbols appear in the outline, breadcrumb, and suggest widget.'));
+
+export const OUTLINE_INTERFACE_SYMBOL_FOREGROUND = registerColor('outlineInterfaceSymbol.foreground', {
+	dark: '#75BEFF',
+	light: '#007ACC',
+	hc: '#75BEFF'
+}, localize('outlineInterfaceSymbol.foreground', 'The foreground color for interface symbols. These symbols appear in the outline, breadcrumb, and suggest widget.'));
+
+export const OUTLINE_VARIABLE_SYMBOL_FOREGROUND = registerColor('outlineVariableSymbol.foreground', {
+	dark: '#75BEFF',
+	light: '#007ACC',
+	hc: '#75BEFF'
+}, localize('outlineVariableSymbol.foreground', 'The foreground color for variable symbols. These symbols appear in the outline, breadcrumb, and suggest widget.'));
```

Comment:
This doesn't look like a complete list of the symbols we have?

Reply:
I only included the ones that have color, the rest default to “outlineDefaultSymbol.foreground”. Are you thinking we’d define a token for each symbol type?

Reply:
You mean those that come with a color by default, right? I feel like folks would expect to be able to change the colour of all icons.

Reply:
Yeah, makes sense